src/app/hero.ts

```diff
@@ -0,0 +1,4 @@
+export class Hero {
+  id:number;
+  name: string;
+}
```

Comment:
Although a class can be used, TypeScript offers the [Interface](https://www.typescriptlang.org/docs/handbook/interfaces.html). Definitely the way to go when defining the _shape_ or _contract_ of the code.

Keep interfaces in mind when you begin your site in ng2.